### PR TITLE
Use Crowdin action for translation sync

### PR DESCRIPTION
**Pull request by denyskon**
Switch from the old self-built action to the official one.

We get:
- config managed inside the repo
- automatic upload when source file changes
- automatic invalidation if source string changes (tested)
- automatic download of new translation files

Tested both upload and download.

### CODE REVIEW

**Review by silverwind — APPROVED** [review]
Lets try it.

**Review by delvh — APPROVED** [review]

File: .github/workflows/cron-translations.yml [hunk 1]
```diff
@@ -4,21 +4,28 @@ on:
   schedule:
     - cron: "7 0 * * *" # every day at 00:07 UTC
   workflow_dispatch:
+  push:
```
Comment (lunny) [inline .github/workflows/cron-translations.yml]: Should we need to check whether the translation updated for every push? I think the schedule can work well.

Comment (denyskon) [reply .github/workflows/cron-translations.yml]: I don't think it is a bad idea, so I can update my translations directly after my PR merged for example 😅 I'm fine either ways.

Comment (techknowlogick) [reply .github/workflows/cron-translations.yml]: We previously had it pull translations on every push, which filled up the commit log with many small commits, so the balance was to sync down new translations once a day. We do have the workflow dispatch step to allow us to run the workflow on demand if there are any we must have immediately.

If that works with you:
```suggestion
```

Comment (denyskon) [reply .github/workflows/cron-translations.yml]: Okay, applied :)

File: .github/workflows/cron-translations.yml [hunk 2]
```diff
@@ -4,21 +4,27 @@ on:
   schedule:
     - cron: "7 0 * * *" # every day at 00:07 UTC
   workflow_dispatch:
+    branches: ["main"]
```
Comment (techknowlogick) [inline .github/workflows/cron-translations.yml]:
```suggestion
```

**Comment by silverwind** [comment]
Seems to work: https://github.com/go-gitea/gitea/commit/bcf3be3a6c2794f7a3841f9d110b14be29327e1a https://github.com/go-gitea/gitea/commit/7eb3ab076549f83fadf4034f77e7794e785725fa.

**Comment by denyskon** [comment]
@silverwind In terms of pull interval, there's nothing different - we just had a manually scheduled test run earlier yesterday to check if it works, otherwise this would have been one commit :)

**Comment by silverwind** [comment]
Ok, then the second commit really was a day's content.

**Pull request merged**